web/themes/custom/lex/scss/molecules/_event_card.scss

```diff
@@ -1,92 +1,104 @@
 .event-card {
```

Comment:
All of this is formatting to 2 spaces. The actual relevant change is at line 92.